src/core/scrollcomponent/BaseScrollComponent.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from "react";
+import type { ScrollViewProps } from "react-native";
```

Comment:
BaseScrollComponent is supposed to be agnostic of react or react native. You can pass this style using `scrollViewProps` to avoid type errors